README.md

```diff
@@ -2,7 +2,7 @@
 
 - [Disclaimer](#disclaimer)
 - [Introduction](#introduction)
-- [Status](#status)
+- [Status](#status) ![](https://github.com/martinvonz/jj/workflows/build/badge.svg)
```

Comment:
This heading is project status, not build status, so I don't think this belongs here. Do you want to instead add a new section with various badges as @arxanas did for https://github.com/arxanas/git-branchless?

Reply:
Do you have any preferences about location? Would you like to be patterned after git-branchless?

Reply:
I don't have much preference, so following git-branchless seems best, unless you have some preference.

Reply:
Also, as I doubt the google-cla bot will respond: every link to sign the Google CLA takes me to a Google sign-in page. Is there a way to sign without having a Google account?

Reply:
I just checked our internal docs and they say that you need to have a Google account to sign it. I can take over the PR if you don't want to register for a Google account.

Reply:
There's probably no need for a table in this case, like in git-branchless. I've seen many projects just put the status badges as the very first line in the README (even above any headers), which looked fine to me.

Reply:
In the updated PR, I moved the tags to the top (but under the project title), and added a number of other badges; feel free to trim out whatever seems excessive.

In the commit, I explicitly release all rights on the changes. If that's insufficient, please feel free to take over the PR. This seems like a lot of conversation considering the scope of the "work." :-)

Reply:
I'd be surprised if Google thinks that's enough, so I just created a copy of this PR as #822. Thanks!